src/pages/about.js

```diff
@@ -48,9 +49,52 @@ module.exports = React.createClass({
         </span>
       );
     }
+
     this.setState({
       aboutCopy: aboutCopy
     });
+
+    var percentIncrement = 10; // We'll track the video progress in 10% increments.
+    if (this.props.test === "video") {
+
+      this.setState({
+        nextPercentDone: percentIncrement,
+        percentIncrement : percentIncrement,
+        videoStarted : false
+      });
+
+      this.refs.videoPlayer.addEventListener("play", this.onVideoStart);
+      this.refs.videoPlayer.addEventListener("timeupdate", this.onVideoTimeupdate);
```

Comment:
Any events you add, you need to ensure you remove inside a `componentWillUnmount` function. Example https://github.com/mozilla/advocacy.mozilla.org/blob/251c8d1429e637998af43464b817087e5c89c45f/src/components/imagetag.js#L28-L34